### PR TITLE
optimize dt_bilateral_slice

### PR DESCRIPTION
**Pull request by ralfbrown**
Tweak OpenMP directive and use copy_pixel instead of three separate assignments to get about a 7% speedup in the function.  The bilateral filter is used in multiple IOPs, so there will be a small speedup in a variety of operations (e.g. this function is 30-40% of the runtime of the Lowpass module, so that gets a 2-3% speed boost).

Passes tests 0015, 0016, 0017.

### CODE REVIEW

File: src/common/bilateral.c [hunk 1]
```diff
@@ -449,6 +448,7 @@ void dt_bilateral_slice_to_output(const dt_bilateral_t *const b, const float *co
       out[index] = MAX(0.0f, out[index] + Lout);
     }
   }
+  fprintf(stderr,"bilat_slice_to %.3f ms\n",1000.0*(dt_get_wtime()-start));
```
Comment (TurboGit) [inline src/common/bilateral.c:451]: All the `fprintf(stderr` have recently been changed, so should be `dt_print(DT_DEBUG_ALWAYS,`.

Comment (TurboGit) [reply src/common/bilateral.c:451]: Let me do the change. I have already written the RELEASE_NOTES entries...

Comment (TurboGit) [reply src/common/bilateral.c:451]: Even easier, this is debug code. I'll remove.

Comment (ralfbrown) [reply src/common/bilateral.c:451]: Oops,  sorry about that -- yeah, that was my timing code to check whether I was actually improving the speed.

**Pull request merged**